core-build/gulp-core-build/src/tasks/copyStaticAssets/copy-static-assets.schema.json

```diff
@@ -5,6 +5,11 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "$schema": {
+      "description": "The metadata to indicate the JSON schema, required by some editors such as VS2015",
```

Comment:
> "The metadata to indicate the JSON schema, required by some editors such as VS2015" [](start = 20, length = 84)

"Part of the JSON Schema standard, this optional keyword  declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting."